packages/sign-client/src/controllers/pendingRequest.ts

```diff
@@ -0,0 +1,10 @@
+import { Store } from "@walletconnect/core";
+import { Logger } from "@walletconnect/logger";
+import { ICore, PendingRequestTypes } from "@walletconnect/types";
+import { REQUEST_CONTEXT, SIGN_CLIENT_STORAGE_PREFIX } from "../constants";
+
+export class PendingRequest extends Store<number, PendingRequestTypes.Struct> {
```

Comment:
Since we both already know this is a gotcha: are we sure that the `PendingRequest` store rehydrates from persistence/`localStorage` as expected without the `getKey` specification?

Should be fine since it has a `topic` field but worth making sure.

Reply:
🤘